### PR TITLE
Automatic code cleanup attempt

### PR DESCRIPTION
**Pull request by Koen1999**

### CODE REVIEW

**Comment by Koen1999** [comment]
Is this something that we might like to do?

File: module/Application/Module.php [hunk 1]
```diff
@@ -9,6 +9,12 @@
 
 namespace Application;
 
+use Application\View\Helper\Acl;
+use Application\View\Helper\FileUrl;
+use Application\View\Helper\Infima;
+use Application\View\Helper\JobCategories;
+use Application\View\Helper\ModuleIsActive;
+use Application\View\Helper\ScriptUrl;
```
Comment (jszanto) [inline module/Application/Module.php]: I'm not really a fan of adding all these imports when each class is only used once here. In my opinion it even makes the code slightly less readable.

**Comment by jszanto** [comment]
This is a good idea, however this will probably cause a lot of merge conflicts with the other currently open PR's. Furthermore if we do such a thing we probably should fix some more of the issues at the same time and make sure that everything is formatted according to PSR-12

File: module/Application/Module.php (hunk 1 above)
Comment (Koen1999) [reply module/Application/Module.php]: I personally think it makes it more readable. But I guess it's rather subjective.

**Pull request closed without merging**